heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -100,13 +100,53 @@
    */
   public static final String TOPOLOGY_SERIALIZER_CLASSNAME = "topology.serializer.classname";
   /**
-   * How many executors to spawn for ackers.
+   * Is the topology running in atleast-once mode?
    * <p>
-   * <p>If this is set to 0, then Heron will immediately ack tuples as soon
+   * <p>If this is set to false, then Heron will immediately ack tuples as soon
    * as they come off the spout, effectively disabling reliability.</p>
+   * @deprecated use {@link #TOPOLOGY_RELIABILITY_MODE} instead.
    */
+  @Deprecated
   public static final String TOPOLOGY_ENABLE_ACKING = "topology.acking";
```

Comment:
I thought there were too, but I don't see it being enabled by using either of these. Is there any other way?